src/_tests_/App.test.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { render } from '@testing-library/react';
+import App from '../components/App';
+
+describe("App", () => {
+  const { asFragment } = render(<App />);
+
+  it("renders correctly", () => {
+    expect(asFragment()).toMatchSnapshot();
+  });
+});
+
+
+
```

Comment:
This could be condensed, asFragment line should be put in as part of the test itself, and the extra clear lines at the end should be removed
```
describe("App", () => {
  it("renders correctly", () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
  });
});
```